caproto/_state.py

```diff
@@ -211,7 +211,7 @@ def __init__(self, channels):
         self.states = {CLIENT: SEND_VERSION_REQUEST, SERVER: IDLE}
         self.channels = channels
 
-    def process_command(self, role, command_type):
+    def process_command_type(self, role, command_type):
```

Comment:
Do you honestly think the @tacaswell would venture into this dark corner of GitHub... ?